docker-compose/mariadb-config/prod/mariadb.cnf

```diff
@@ -0,0 +1,58 @@
+[mariadb]
+
+port                  = 3306
+bind-address          = 0.0.0.0
+
+# Fine tuning
+max_connections         = 300
+connect_timeout         = 5
+wait_timeout            = 600
+max_allowed_packet      = 32M
+thread_cache_size       = 128
+sort_buffer_size        = 128M
+bulk_insert_buffer_size = 128M
+tmp_table_size          = 32M
+max_heap_table_size     = 32M
+skip-name-resolve       = 1
+innodb_log_file_size    = 2G
+
+# Logging
+# warning log-basename is defined later for replication
+# some log option may be ignored.
+# see: https://mariadb.com/kb/en/mysqld-options/#-log-basename
+# log_error = /var/log/mysql/error.log
+slow_query_log
+slow_query_log_file    = /var/log/mysql/mariadb-slow.log
```

Comment:
For forward compatibility and consistency these where renamed in 10.11
```
log_slow_query
log_slow_query_file = mariadb-slow.log
```

There's no harm in this just being in mariadb-slow.log (defaults to datadir). It eliminates the need to have a log volume.

note however there is no internal mariadb mechanism to rotate this file automaticly and a logrotate service doesn't run in the container. Its still possible to do it manually moving/removing the file and then `FLUSH SLOW LOG`. It normally won't be a problem unless long_query_time is slow slow it catches most queries.

Reply:
So, if I understand this right, writing the file to `datadir` makes it accessible by this volume mount 
`- ./mariadb:/var/lib/mysql:rw`

right?